game.cpp

```diff
@@ -86,11 +93,20 @@ bool Game::Run() {
     graphics_.DrawNextFrame();
     perf.StopTimer("draw");
 
+    LimitFrames();
     perf.StopTimer("game_loop");
     perf.ReportResults();
   }
 }
 
+void Game::LimitFrames() {
+  int current_time_ms = SDL_GetTicks();
+  int elapsed_time_ms = current_time_ms - frame_start_time_ms;
+  if (elapsed_time_ms < constants::kMaxFrameTimeMs) {
+    usleep(1000 * (constants::kMaxFrameTimeMs - elapsed_time_ms));
```

Comment:
I found out there is a function called SDL_Delay that pauses for the given number of ms. Let's use that here instead.